Public/Src/Deployment/buildXL.dsc

```diff
@@ -31,24 +31,16 @@ namespace BuildXL {
             ...addIfLazy(qualifier.targetRuntime !== "osx-x64", () => [{
                 subfolder: r`tools`,
                 contents: [

```

Comment:
>contents: [ [](start = 16, length = 11)

I might be missing something here. we used to place two dirs - 'bxp' and 'bxp-server'. Now we are placing only 'bxp-server'